zerver/decorator.py

```diff
@@ -612,8 +613,8 @@ def to_utc_datetime(timestamp: Text) -> datetime.datetime:
     return timestamp_to_datetime(float(timestamp))
 
 WrapperT = Callable[[Callable[..., ReturnT]], Callable[..., ReturnT]]
-def statsd_increment(counter, val=1):
-    # type: (Text, int) -> Callable[[Callable[..., ReturnT]], Callable[..., ReturnT]]
+def statsd_increment(counter: Text,
+                     val: int=1) -> Callable[[Callable[..., ReturnT]], Callable[..., ReturnT]]:
```

Comment:
We should have the arguments on the same line and probably have the return type on a different line